apps/guardian-ui/src/setup/FederationSetup.tsx

```diff
@@ -118,19 +118,17 @@ export const FederationSetup: React.FC = () => {
           </Button>
         )}
         {title && (
-          <Heading size='md' fontWeight='medium'>
+          <Heading size={['sm', 'md']} fontWeight={['medium']}>
```

Comment:
Nit, if it's not a responsive value you might as well just keep it a string

Reply:
fixed